pkg/ssh/shell/cleanupDevnet.sh

```diff
@@ -0,0 +1,8 @@
+#!/ust/bin/env bash
```

Comment:
can we call this setupDevnet.sh or startDevnet.sh or initDevnet.sh

Reply:
done